src/components/Panel.ts

```diff
@@ -454,7 +470,14 @@ class Panel implements FlickingPanel {
     }
   }
 
-  public setElement(element: HTMLElement): void {
+  public setElement(element: HTMLElement | null): void {
+    if (!element) {
+      return;
+    }
+    if (!this.element) {
+      this.initElement(element);
```

Comment:
it looks confused that `initElement` and `setElement` each other. It could be combine to `setElement` and remove `initElement`.